README.md

```diff
@@ -1,24 +1,159 @@
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- **[ Data model]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema.sql](./schema.sql) to create all tables.
+- Use [data.sql](./data.sql) to populate tables with sample data.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+`install postgresql.`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`cd my-folder git clone https://github.com/mumo-esther/Clinic-DB.git`
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+`sh bin/rails test test/models/article_test.rb`
+
+### Deployment
+
+You can deploy this project using:
+`render`, `heroku`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Esther Mawioo**
+
+- [Github](https://github.com/mumo-esther/mumo-esther)
+- [Linkedin](https://www.linkedin.com/in/esther-mawioo-58b636225/)
+- [Twitter](https://twitter.com/EstherMawioo)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add more relational tables for the Blog App]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- Hat tip to everyone whose code was used.
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star and you can inbox me from my profile if you need such a project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse.
+
+I would like to thank my coding partners and my family for their support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I noticed that you are missing the `LICENCE` file in your directory and this is the reason we are getting this error :point_down: kindly add the `LICENSE` file in your directory then Import it in your readme :wink: 
![Screenshot from 2023-06-08 15-17-33](https://github.com/mumo-esther/Blog-App/assets/88894525/31829369-b2a3-471e-aa6a-15111e867d04)